components/CancellationList.vue

```diff
@@ -63,16 +66,18 @@
               text
               @click="sendReport"
             >
-              Report
+              {{ $t("cancelList.report.submitReport") }}
+            </v-btn>
+            <v-btn color="secondary" text @click="cancelReport">
```

Comment:
In general, cancel buttons should be secondary as they are not the primary action

https://uxplanet.org/primary-secondary-action-buttons-c16df9b36150

Reply:
+1